debian/changelog

```diff
@@ -1,3 +1,16 @@
+authd (0.5.0) questing; urgency=medium
+
+  * Use SQLite instead of bbolt
+  * Use lowercase usernames
+  * Fix races in the PAM module.
+  * Do not support device authentication mode in polkit.
+  * Don't overwrite shell when updating user entry
+  * Improve warning logged when home directory has incorrect owner/group
+  * Support logging to the journal with priority "notice"
+  * More useful error messages
```

Comment:
And safer? Given we've PR hiding some secrets in debug mode

Reply:
I added `Hide secrets in debug messages`